infra/secretsmanager/main.tf

```diff
@@ -0,0 +1,50 @@
+locals {
+  file_secret_name   = "${var.DSS_SECRETS_STORE}/${var.DSS_DEPLOYMENT_STAGE}/${var.DSS_FILE_VERIFY_SECRETS_NAME}"
+  blob_secret_name   = "${var.DSS_SECRETS_STORE}/${var.DSS_DEPLOYMENT_STAGE}/${var.DSS_BLOB_VERIFY_SECRETS_NAME}"
+  bundle_secret_name = "${var.DSS_SECRETS_STORE}/${var.DSS_DEPLOYMENT_STAGE}/${var.DSS_BUNDLE_VERIFY_SECRETS_NAME}"
+}
+
+resource "aws_secretsmanager_secret" "files_last_verified" {
```

Comment:
please apply resource tagging if available, see `infra/buckets/s3.tf` for help.

Reply:
Done!